### PR TITLE
Add PR dependencies feed for Taef Nuget

### PR DESCRIPTION
**Pull request by kythant**
Add PR dependencies feed for Taef Nuget

### CODE REVIEW

File: build/Helix/NuGet.config [hunk 1]
```diff
@@ -19,7 +19,8 @@
     <clear />
     <add key="NuGet official package source" value="https://nuget.org/api/v3/" />
     <add key="dotnetfeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="WindowsES-External" value="https://microsoft.pkgs.visualstudio.com/_packaging/WindowsES-External/nuget/v3/index.json" />	
+    <add key="WindowsES-External" value="https://microsoft.pkgs.visualstudio.com/_packaging/WindowsES-External/nuget/v3/index.json" />
```
Comment (huichen123) [inline build/Helix/NuGet.config:22]: do we still need this? also we can remove those authenticate steps

Comment (kythant) [reply build/Helix/NuGet.config:22]: Yea we don't

**Review by huichen123 — APPROVED** [review]

**Review by riverar — CHANGES_REQUESTED** [review]
This still fails in the public build. CppTest still has the bad reference.

From the referenced issue:
https://github.com/microsoft/ProjectReunion/blob/93e786dc7461e41ddd4afc2476a4d1cf765a6bdd/test/CppTest/CppTest.vcxproj#L342

**Comment by kythant** [comment]
> This still fails in the public build. CppTest still has the bad reference.
> 
> From the referenced issue:
> https://github.com/microsoft/ProjectReunion/blob/93e786dc7461e41ddd4afc2476a4d1cf765a6bdd/test/CppTest/CppTest.vcxproj#L342

Are you absolutely sure? Nothing has changed on that end. THe same version was uploaded.

**Comment by riverar** [comment]
@kythant The line highlighted performs the Error action if the `..\..\packages\Microsoft.Taef.10.58.210222006-develop\build\Microsoft.Taef.targets` file does not exist. Because I don't have access to that package, this will always fail when compiling CppTest.

This should be easy to repro on an external machine. Happy to provide RDP to a test machine if you need one for testing this PR.

**Review by BenJKuhn — APPROVED** [review]

**Comment by riverar** [comment]
Looks like the Taef package is now available; build passes. 👍

**Comment by kythant** [comment]
> @kythant The line highlighted performs the Error action if the `..\..\packages\Microsoft.Taef.10.58.210222006-develop\build\Microsoft.Taef.targets` file does not exist. Because I don't have access to that package, this will always fail when compiling CppTest.
> 
> This should be easy to repro on an external machine. Happy to provide RDP to a test machine if you need one for testing this PR.

Did you restore the package? I've added it to PR-dependencies so you can pull the package from it.

**Comment by YexuanXiao** [comment]
It looks like the package has been removed from the source at the moment, so it's still not possible to build the project.

**Pull request merged**